Automattic/jetpack
# Protect: Fix in-progress fixer indicator

## Description
Ensure the in-progress spinner renders when a threat fixer has that status.

## Proposed changes:

* Fixes the existing redux state for `threatsAreFixing`
* Disable the `Ignore`/`Fix threat` buttons when a threat fixer is `in-progress`

### Other information:

## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1226

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Create a JN site
* Install the Jetpack Beta plugin and enable Protect using this PR as the feature branch
* Upgrade your plan
* Install a known threat that is fixable and run the scan to detect it
* Queue the fixer for the threat and ensure that for the duration that the threat has an `in-progress` fixer status:
   * A loading `Spinner` component is render in the `Auto-fix` column
   * The `Ignore`/`Fix threat` buttons are disable

## Screenshots
![Screen Shot on 2024-06-26 at 13-14-30](https://github.com/Automattic/jetpack/assets/43220201/757ceff9-a518-438b-8d76-e8b2ce7388d8)
